Unsorted/24300444_172cb9c.txt

```diff
@@ -10,4 +10,7 @@
 		武器の手入れは<end_line>
 		なまけちゃいかんな<end_line>
 	</sjis>
+	<ascii>
+		Don't forget to repair your weapons<end_line>
```

Comment:
Here the chief is saying something the lines of "dont be lazy/careless, take care of your weapons". This could also be split into two lines as adding a period would make it too long.

Reply:
Yes, what he says is exactly as you mentioned. I interpreted as one of those npc dialogues we see in teh beginning of games that teaches about the game actions/stuff we can or should do in the game , that's why I phrased that way. But you are right, I changed the meaning of it, I'll rephrase it 👍 .

Reply:
gotta mention too, in-game mechanics are, most often than not, thaught at the forge, it has a special box which has "INFORMATION" as text. One time I have seen the contrary is when you just get your partner to team up when searching for the kicker berry, There it teaches you about how you can change your partner mid-battle

Reply:
I changed to : 
> Don't be careless,<end_line>
> take care of your weapons.<end_line>

commit:  2796089 3e8baaf